components/cards/card.css

```diff
@@ -0,0 +1,306 @@
+.card-verticle {
+  max-width: 300px;
+  width: 100%;
+  height: max-content;
+  background-color: var(--card-bg);
+  position: relative;
+  box-shadow: var(--dark-shadow);
+}
+
+.image-container {
+  position: relative;
+  width: fit-content;
+}
+
+.card-image {
+  width: 100%;
+  height: 240px;
+
+  overflow: none;
+}
```

Comment:
`overflow` does not have `none` value, Maybe you need `hidden` here

Reply:
changed that to hidden.